test/simulator/models/mass.jl

```diff
@@ -1,6 +1,6 @@
 using Test
 using FlightMechanics.Models
-
+using LinearAlgebra
```

Comment:
Again, I think this was working thanks to another import in a previous include. I will check the right way of doing this. Let me know if you have found a standard

Reply:
i know it was working fine when running tests from the runtests.jl file but when i tried to run mass.jl test file it was showing undefined variable error thats why added this.